services/121-service/src/notifications/twilio.mock.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import { Injectable } from '@nestjs/common';
+import {
+  TwilioCallsCreateDto,
+  TwilioMessagesCreateDto,
+  TwilioValidateRequestDto,
+} from './twilio.dto';
+
+@Injectable()
+export class TwilioClientMock {
+  public messages;
+  public lookups;
+  public calls;
+  public constructor() {
+    this.messages = new this.Messages();
+    this.lookups = new this.Lookups();
+    this.calls = new this.Calls();
+  }
+
+  public Messages = class {
+    public async create(
+      twilioMessagesCreateDto: TwilioMessagesCreateDto,
+    ): Promise<object> {
+      return {
+        body: twilioMessagesCreateDto.body,
+        numSegments: twilioMessagesCreateDto.mediaUrl ? '1' : '0',
+        direction: 'outbound-api',
+        from: twilioMessagesCreateDto.from,
+        to: twilioMessagesCreateDto.to,
+        dateUpdated: new Date(),
+        price: null,
+        errorMessage: null,
+        uri:
+          '/2010-04-01/Accounts/ACc6b5fa2276ff6b64cdbed0099fd9bc0b/Messages/SM2659aa9303394e01b30f33cbd92c4cf3.json',
+        accountSid: 'ACc6b5fa2276ff6b64cdbed0099fd9bc0b',
```

Comment:
These values are not super-sensitive, but they don't belong in code.
You can use their ENV-versions instead...

```suggestion
          `/2010-04-01/Accounts/${process.env.TWILIO_SID}/Messages/SM2659aa9303394e01b30f33cbd92c4cf3.json`,
        accountSid: process.env.TWILIO_SID,
```